hack/testing/run_daemon_cri_integration.sh

```diff
@@ -127,14 +132,20 @@ integration::run_daemon_cri_test_cases() {
   sleep 5
 }
 
-main() {
-  local cmd flags
+integration::run_cri_test(){
+  local cri_runtime cmd flags coverage_profile
+  cri_runtime=$1
+
+  # daemon cri integration coverage profile
+  coverage_profile="${REPO_BASE}/coverage/integration_daemon_cri_${cri_runtime}_profile.out"
+  echo "${coverage_profile}"
```

Comment:
please remove this line. we can use `bash -x` to check the value~

Reply:
Done.